client/src/index.html

```diff
@@ -8,7 +8,7 @@
   <meta property="og:description" content="¡Conoce la ubicación del patrimonio de Medellín!" />
   <meta property="fb:app_id" content="1124058024274759" />
   <meta property="og:image" content="../images/facebook-image.png" />
-
```

Comment:
This change is in your fb/twitter branch I believe (and seems more related to those changes)